### PR TITLE
fix: Two BUG 

### PR DESCRIPTION
**Pull request by snowsi**
1. Encrypted documents in. 7z format cannot delete locked files and folders
    Step: Double click to open a. 7z encrypted document, select a file, and right-click to delete it.

    Because the rm command is executed after the delete command is executed, the error output of the delete command is 
    released, and the_ command_ 7z_ handle_ Error cannot be caught in error

2. Encrypted documents in all formats cannot be successfully pasted to the specified directory after copying and cutting 
    encrypted files and folders
    Step: Double click to open the. 7z encrypted document, select a file, right-click to copy, cut, and then right-click to paste it 
    into the package folder

    Because the error information of the last command was not cleared during false loading, the error information of the last 
    command was processed during error processing

### CODE REVIEW

File: src/fr-command-7z.c [hunk 1]
```diff
@@ -393,13 +393,30 @@ fr_command_7z_add (FrCommand     *comm,
 	fr_process_end_command (comm->process);
 }
 
+static void
+process_line__delete (char     *line,
+		      gpointer  data)
+{
+	FrCommand *comm = FR_COMMAND (data);
```
Comment (zhuyaliang) [inline src/fr-command-7z.c:400]: Unused variable 'comm'

File: src/fr-window.c [hunk 2]
```diff
@@ -6317,14 +6317,34 @@ fr_window_archive_add_dropped_items (FrWindow *window,
 				      window->priv->volume_size);
 }
 
+static void
+remove_data_free (GList *rdata)
+{
+	g_list_free_full(rdata, g_free);
```
Comment (zhuyaliang) [inline src/fr-window.c:6323]: Add space

Comment (lukefromdc) [reply src/fr-window.c:6323]: Another trivial fix, then this conversation can be resolved and this should be mergable

File: src/fr-command-7z.c (hunk 1 above)
Comment (lukefromdc) [reply src/fr-command-7z.c:400]: This should be trivial to fix

File: src/fr-window.c (hunk 2 above)
Comment (zhuyaliang) [reply src/fr-window.c:6323]: @lukefromdc If the author still doesn't reply after two days, I will merge this PR

**Comment by zhuyaliang** [comment]
@lukefromdc Any further questions? I'm going to merge

**Comment by lukefromdc** [comment]
No further questions, feel free to proceed

**Pull request merged**